### ISSUE TITLE
Ansible 2.3 can't start tests

### ISSUE THREAD

**Issue by enkov** [philpep/testinfra#194]
Hello. After upgrade to ansible 2.3 starting tests fails with error:
```
============================= test session starts ==============================
platform linux2 -- Python 2.7.12, pytest-3.0.7, py-1.4.33, pluggy-0.4.0
rootdir: /home/enkov/WORK1/playbooks/roles/foo, inifile:
plugins: testinfra-1.5.4
collected 0 items / 1 errorss

==================================== ERRORS ====================================
____________________ ERROR collecting tests/test_default.py ____________________
tests/test_default.py:4: in <module>
    '.molecule/ansible_inventory').get_hosts('all')
/usr/local/lib/python2.7/dist-packages/testinfra/utils/ansible_runner.py:141: in __init__
    self.cli = ansible.cli.CLI(None)
E   TypeError: Can't instantiate abstract class CLI with abstract methods parse, run
============================ pytest-warning summary ============================
WP1 None Module already imported so can not be re-written: testinfra
!!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!
================== 1 pytest-warnings, 1 error in 0.20 seconds ==================
ERROR: 

  RAN: '/usr/local/bin/testinfra tests/test_default.py --connection=docker'

  STDOUT:


  STDERR:
```
With ansible version 2.2.1 everything works fine.

**Comment by philpep** [comment]
Issue should be fixed in (just released) version 1.5.5.

Thanks for reporting !

**Comment by enkov** [comment]
I tested and it works fine.
Thank you for very fast fix.
